src/sql/resolver/dml/ob_insert_resolver.h

```diff
@@ -119,7 +119,7 @@ class ObInsertResolver : public ObDMLResolver {
   {
     is_oracle_tmp_table_ = is_temp_table;
   }
-  common::hash::ObPlacementHashSet<uint64_t>& get_insert_column_ids()
+  common::hash::ObPlacementHashSet<uint64_t, 8209>& get_insert_column_ids()
```

Comment:
add comment tells why is 8209

Reply:
In the original implementation, the default length of ObPlacementHashSet is 1039, which is the first prime number greater than 512. After changing the max column to 4096, I think changing it to 8209 (the first prime number greater than 4096) is a good choice.